cost/volumes/uav_policy.pt

```diff
@@ -0,0 +1,288 @@
+      }
+    })
+    
+    my_vol["href"] = vol_href
+    my_vol["cloud_href"] = vol_cloud_href 
+    my_vol["cloud_name"] = cloud_hash[vol_cloud_href]["cloud_name"]
+    
+    // Now use that cloud hash built above to see if this volume is in Azure/RM
+    if (cloud_hash[vol_cloud_href]["cloud_type"] == "azure_v2") {  
+      // AzureRM - we care
+      if (my_vol["pg_id"]) {
+        my_vol["azure_disk_type"] = "Unmanaged"
+        my_vol["azure_storage_type"] = pg_hash[my_vol["pg_id"]]["account_type"]
+        my_vol["azure_storage_account_name"] = pg_hash[my_vol["pg_id"]]["name"]
+      } else if (my_vol["volume_type"]) {
+        my_vol["azure_disk_type"] = "Managed"
+        my_vol["azure_storage_type"] = my_vol["volume_type"]
+        my_vol["azure_storage_account_name"] = "N/A"
+      } else {
+        my_vol["azure_disk_type"] = "Unknown"
+        my_vol["azure_storage_type"] = "Unknown"
+        my_vol["azure_storage_account_name"] = "Unknown"
+      }
+    } else if (cloud_hash[vol_cloud_href]["cloud_type"] == "azure") {   
+      // old Azure - we also care
+      my_vol["azure_disk_type"] = "Unmanaged"
+      my_vol["azure_storage_type"] = "Classic"
+      my_vol["azure_storage_account_name"] = pg_hash[my_vol["pg_id"]]["name"]
+    } else {
+      // Other clouds - we do not care
+      my_vol["azure_disk_type"] = "N/A"
+      my_vol["azure_storage_type"] = "N/A"
+      my_vol["azure_storage_account_name"] = "N/A"
+    }
+    
+    // Calculate the age and add the information to the hash for the unattached volume.
+    // This is used in the reporting to tell the user the age of the unattachedness.
+    // Get the parts of the updated_at date found in the hash
+    var updated_at_parts = vol.updated_at.split(" ")
+    // chop up the yyyy/mm/dd into its parts
+    var date_parts = updated_at_parts[0].split("/")
+
+    // Create a Date object based on the yyyy, mm, dd
+    // NOTE: months in javascript goes 0-11 so subtract one from the mm bit 
+    var updated_at_date = new Date(date_parts[0], (Number(date_parts[1]) - 1), date_parts[2])
+    var updated_at_ms = updated_at_date.getTime()
+    var difference_ms = Math.abs(today_ms - updated_at_ms)
+    var number_of_days_old = Math.round(difference_ms/ONE_DAY)
+
+    // Add a computed days old item to the hash
+    my_vol["age"] = number_of_days_old
+
+    // Push the volume hash element to the unattach_volumes array
+    unattached_volumes.push(my_vol)  
+  }
+}
+EOS
+end
+	
+policy "pol_unattached_volumes" do
+  validate_each $ds_unattached_volumes do
+	# Go through the unattached volumes that were found by the script-based datasource above
+	# and check if not older than specified number of days.
+	# If the check fails, it'll flag the volume
+	check le(val(item, "age"), $param_days_old)
+
+	escalate $esc_unattached_volumes
+
+	summary_template "Unattached Volumes"
+
+#    	detail_template <<-EOS
+#Unattached Volume Report for Account: {{ rs_project_name }} (ID: {{ rs_project_id }})
```

Comment:
should this section be removed since it's not used.

Reply:
Yes. Removed